modules/clients/init.sh

```diff
@@ -56,6 +56,7 @@ function setup_aws_logs_agent() {
 setup_aws_logs_agent || echo "Failed to setup AWS logs agent"
 
 yum -y install pip || true
+yum -y install kernel-devel || true
```

Comment:
why do we need it?

Reply:
`^@error: Driver command failed with error code 1
Error: Unable to find a match: kernel-devel-6.1.106-116.188.amzn2023.x86_64
Amazon Linux 2023 repository                    1.3 kB/s | 243  B     00:00    
Errors during downloading metadata for repository 'amazonlinux':
  - Status code: 403 for https://al2023-repos-eu-west-1-de612dc2.s3.dualstack.eu-west-1.amazonaws.com/core/mirrors/2023/x86_64/mirror.list (IP: 52.92.32.186)
  - Status code: 403 for https://al2023-repos-eu-west-1-de612dc2.s3.dualstack.eu-west-1.amazonaws.com/core/mirrors/2023/x86_64/mirror.list (IP: 3.5.65.151)
Error: Failed to download metadata for repo 'amazonlinux': Cannot prepare internal mirrorlist: Status code: 403 for https://al2023-repos-eu-west-1-de612dc2.s3.dualstack.eu-west-1.amazonaws.com/core/mirrors/2023/x86_64/mirror.list (IP: 52.92.32.186)
Ignoring repositories: amazonlinux
Last metadata expiration check: 0:01:23 ago on Sun Sep  8 07:35:05 2024.
No match for argument: kernel-devel-6.1.106-116.188.amzn2023.x86_64
Error: Unable to find a match: kernel-devel-6.1.106-116.188.amzn2023.x86_64
:
`